lib/mumuki/domain/submission/base.rb

```diff
@@ -9,6 +9,7 @@ class Mumuki::Domain::Submission::Base
                 :submission_id, :queries, :query_results, :manual_evaluation_comment]
 
   attr_accessor *ATTRIBUTES
+  attr_accessor :client_result
```

Comment:
Are we planning on using it in a submission type other than a `solution`?

Reply:
I am not planning that, but that would be definitively a good idea to implement in the future. Playground and interactive exercises may benefit of this feature as well.